assets/js/components/settings/SettingsApp.test.js

```diff
@@ -41,14 +41,48 @@ describe( 'SettingsApp', () => {
 		registry.dispatch( CORE_USER ).receiveGetAuthentication( { needsReauthentication: false } );
 		registry.dispatch( CORE_USER ).receiveConnectURL( 'test-url' );
 
-		provideModules( registry, [ {
-			slug: 'analytics',
-			name: 'Analytics',
-			active: true,
-			connected: true,
-			setupComplete: true,
-			SettingsEditComponent: () => <div data-testid="edit-component">edit</div>,
-		} ] );
+		provideModules( registry, [
+			{
+				slug: 'analytics',
+				name: 'Analytics',
+				active: true,
+				connected: true,
+				setupComplete: true,
+				SettingsEditComponent: () => <div data-testid="edit-component">edit</div>,
```

Comment:
@aaemnnosttv If I remove the `SettingsEditComponent` here, one of the test (`should change location hash & DOM correctly when module is being edited`) will fail since it's looking for the element having the `edit-component` id.

Reply:
I see this works but it's kind of "doing it wrong" and there are other side effects that we don't want here from doing it properly via `provideModuleRegistrations` so I suppose it's okay for now. Ideally, I think the tests here wouldn't use real modules at all, which would solve all the problems but we don't need to rewrite them so let's not 😄